jekyll/_cci2/runner-manually-upgrading-launch-agent.adoc

```diff
@@ -0,0 +1,141 @@
+curl -sSL "${base_url}/${agent_version}/checksums.txt" -o checksums.txt
+mkdir -p "${platform}"
+file="$(grep -F "${platform}" checksums.txt | cut -d ' ' -f 2 | sed 's/^.//')"
+curl --compressed -L "${base_url}/${agent_version}/${file}" -o "${file}"
+grep "${file}" checksums.txt | sha256sum --check || echo "Failed to download the CircleCI machine runner binary, please try again"
+chmod +x $file
+```
+
+[#manually-download-windows-binary]
+=== Manually downloading the windows machine runner
+
+```shell
+# Download the new version of the CircleCI Agent manually on Windows with PowerShell
+$Env:platform = "windows/amd64"
+mkdir $Env:platform
+$Env:baseURL =  "https://circleci-binary-releases.s3.amazonaws.com/circleci-launch-agent"
+$Env:agentVersion = $(curl "$Env:baseURL/release.txt")
+$Env:agentVersion = "$Env:agentVersion".Trim()
+curl "$Env:baseURL/$Env:agentVersion/checksums.txt" -o checksums.txt
+$Env:fileName = $(Select-String $Env:platform checksums.txt | cut -d ' ' -f 2 | sed 's/^.//')
+$Env:fileName = $Env:fileName.Trim()
+curl "$Env:baseURL/$Env:agentVersion/$Env:fileName" -o $Env:fileName
+$Env:fileHash = $(Get-FileHash $Env:fileName -Algorithm "SHA256" | select -expand "Hash")
+$Env:expectedHash = "$(grep $Env:filename checksums.txt)".Split(" ")[0]
+if ($Env:expectedHash -ne $Env:fileHash) { Write-Output "CircleCI machine runner binary download failed, please try again"}else{Write-Output "CircleCI machine runner binary successfully downloaded to $CWD/$Env:fileName"}
+```
+
+== Manually replacing the machine runner binary
+Once the new binary has been successfully downloaded, the machine runner instance will need to be stopped, have the currently installed binary overwritten with the newly downloaded binary, and then be started again.
+
+[#manually-replace-linux-binary]
+=== Linux 
+```shell
+sudo systemctl stop circleci.service
+# Replace the current binary with the newly downloaded binary, by default the machine runner binary is installed to `/opt/circleci/circleci-launch-agent`
+sudo systemctl start circleci.service
+```
+
+[#manually-replace=macos-binary]
+=== macOS
+```shell
+sudo launchctl stop '/Library/LaunchDaemons/com.circleci.runner.plist'
+# Replace the current binary with the newly downloaded binary, by default the machine runner binary is installed to `/opt/circleci/circleci-launch-agent`
+sudo launchctl start '/Library/LaunchDaemons/com.circleci.runner.plist'
+```
+
+[#manually-replace-windows-binary]
+=== Windows
+```shell
+Stop-ScheduledTask -TaskName "CircleCI Launch Agent"
+# Replace the current binary with the newly downloaded binary, this will require renaming or deleting the existing binary
+Start-ScheduledTask -TaskName "CircleCI Launch Agent"
+```
+
+If process management is not being used to manage the machine runner, the process will need to be manually stopped using `kill -9` on macOS and Linux systems, or the Task Manager on Windows systems, and then manually restarted.  
```

Comment:
```suggestion
NOTE: If process management is not used to manage the machine runner, the process will need to be manually stopped using `kill -9` on macOS and Linux systems, or the Task Manager on Windows systems, and then manually restarted.  
```
I might also suggest moving this note earlier in this section, perhaps after the first paragraph "Once the new binary has been download..."